src/zope/paste/tests.py

```diff
@@ -17,20 +17,23 @@
 import doctest
 import gc
 import os
-import signal
-import subprocess
-import tempfile
 import shutil
+import sys
+import tempfile
 import unittest
 
 import paste.deploy
 from waitress.server import WSGIServer
 
 try:
-    from urllib2 import urlopen
-except ImportError:
-    # Py3: The location moved.
     from urllib.request import urlopen
+except ImportError:
+    # PY2
+    from urllib2 import urlopen
+
+# let every Python version run on its own port to allow parallel runs:
+PORT = 8765 + 10 * sys.version_info.major + sys.version_info.minor
```

Comment:
This formula will make Python 3.10 clash with 4.0, assuming there's ever a 4.0.

I suppose we can fix it then.

Reply:
👍